presto-druid/src/test/java/io/prestosql/plugin/druid/DruidQueryRunner.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.druid;
+
+import io.prestosql.Session;
+import io.prestosql.metadata.QualifiedObjectName;
+import io.prestosql.plugin.tpch.TpchPlugin;
+import io.prestosql.testing.DistributedQueryRunner;
+import io.prestosql.testing.MaterializedRow;
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.tpch.TpchTable;
+import org.intellij.lang.annotations.Language;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static io.airlift.testing.Closeables.closeAllSuppress;
+import static io.prestosql.plugin.druid.DruidJdbcClient.DRUID_CATALOG;
+import static io.prestosql.plugin.druid.DruidJdbcClient.DRUID_SCHEMA;
+import static io.prestosql.plugin.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+import static io.prestosql.testing.TestingSession.testSessionBuilder;
+import static java.lang.String.format;
+
+public class DruidQueryRunner
+{
+    private DruidQueryRunner() {}
+
+    public static QueryRunner createDruidQueryRunnerTpch(TestingDruidServer testingDruidServer, TpchTable table)
+            throws Exception
+    {
+        DistributedQueryRunner queryRunner = null;
+        try {
+            queryRunner = DistributedQueryRunner.builder(createSession()).setNodeCount(3).build();
+            queryRunner.installPlugin(new TpchPlugin());
+            queryRunner.createCatalog("tpch", "tpch");
+
+            Map<String, String> connectorProperties = new HashMap<>();
+            connectorProperties.putIfAbsent("connection-url", testingDruidServer.getJdbcUrl());
+            queryRunner.installPlugin(new DruidJdbcPlugin());
+            queryRunner.createCatalog(DRUID_CATALOG, DRUID_SCHEMA, connectorProperties);
```

Comment:
```suggestion
            queryRunner.createCatalog("druid", "druid", connectorProperties);
```

- DruidJdbcClient. DRUID_CATALOG describes fixed catalog on Druid side; on Presto side we could choose any catalog name we want (like "sales_data"), and we just chose to name the catalog "druid" by convention
- DruidJdbcClient. DRUID_SCHEMA describes fixed schema on Druid side; here this is connector name (which incidentally is also "druid")